config/routes.rb

```diff
@@ -48,6 +48,9 @@
     end
   end
 
+  get '/account/edit_address', :to => 'spree/users#edit_address'
```

Comment:
hi @mkllnk, after you think the code is good, then can you help me to deploy it on staging and let @sstead have a look on Wednesday?